kube-client/src/client/body.rs

```diff
@@ -44,6 +44,11 @@ impl Body {
         Body::new(Kind::Wrap(body.map_err(Into::into).boxed_unsync()))
     }
 
+    /// Collect all the pieces of the body
+    pub async fn collect_bytes(self) -> Result<Bytes, crate::Error> {
+        Ok(<Self as BodyExt>::collect(self).await?.to_bytes())
```

Comment:
I see that `BodyExt` is already imported, can't we just do:

```suggestion
        Ok(self.collect().await?.to_bytes())
```

I ran a cargo check and it seems to compile. I might be missing something but we might not need to disambiguate here using the fully qualified syntax?

Reply:
huh, you are right. not sure what i did wrong to put that there in the first place. thanks!